webapp/templates/purl-create.xhtml

```diff
@@ -20,30 +14,52 @@
+            <!-- These are not used by the template but are required for validation when the resource is created -->
+            <input type="hidden" property="calli:copy" content="?copy" value="{?copy}"/>
+            <input type="hidden" property="calli:canonical" content="?canonical" value="{?canonical}"/>
+            <input type="hidden" property="calli:alternate" content="?alternate" value="{?alternate}"/>
+            <input type="hidden" property="calli:describedby" content="?describedby" value="{?describedby}"/>
+            <input type="hidden" property="calli:resides" content="?resides" value="{?resides}"/>
+            <input type="hidden" property="calli:moved" content="?moved" value="{?moved}"/>
+            <input type="hidden" property="calli:missing" content="?missing" value="{?missing}"/>
+            <input type="hidden" property="calli:gone" content="?gone" value="{?gone}"/>
+            <input type="hidden" property="calli:illegal" content="?illegal" value="{?illegal}"/>
+        </div>
+        <input id="cacheProperty" type="hidden" property="calli:cacheControl" onchange="calli.updateProperty(event, 'calli:cacheControl')"/>
         <div class="row">
             <div class="col-sm-8">
-                <fieldset>
-                    <div class="form-group">
-                        <label for="local">Local name</label>
-                        <div>
-                            <input type="text" id="local" value="{rdfs:label}" class="form-control" required="required" autofocus="autofocus"
-                                onchange="calli.updateProperty(event, 'rdfs:label')" />
-                        </div>
-                    </div>
-                </fieldset>
                 <div class="row">
-                    <fieldset class="col-sm-6">
-                        <div class="form-group">
+                    <fieldset>
+                        <div class="form-group col-sm-12">
+                            <label for="local">Local name</label>
+                            <div>
+                                <input type="text" id="local" value="{rdfs:label}" class="form-control" required="required" autofocus="autofocus"
+                                    onchange="calli.updateProperty(event, 'rdfs:label')" />
+                            </div>
+                        </div>
+                        <div class="form-group col-sm-12">
                             <label for="comment">Comment</label>
                             <div>
                                 <textarea id="comment" class="form-control"
                                     onchange="calli.updateProperty(event, 'rdfs:comment')">{rdfs:comment}</textarea>
                             </div>
                         </div>
-                        <div class="form-group">
-                            <label for="type">GET status</label>
+                    </fieldset>
+                </div>
+                <div class="row">
```

Comment:
Close this row after the target field and start another row for status and cache control